src/Pages/MyCart/MyCart.js

```diff
@@ -0,0 +1,160 @@
+import react, { Component } from "react";
+import SelleProduct from "./SelleProduct";
+import configData from "../../config.json";
+import "./MyCart.scss";
+
+class MyCart extends Component {
+  constructor() {
+    super();
+    this.state = { allChekced: true };
+  }
+  componentDidMount() {
+    fetch(configData.MYCART, {
+      headers: {
+        Authorization: localStorage.getItem("key"),
+      },
+    })
+      .then((res) => res.json())
+      .then((res) => this.setState({ myCartData: res.data }));
+  }
+
+  changeQuantity = (e, product) => {
+    const { value, dataset } = e.target;
+    const myCartData = [];
+    this.state.myCartData.forEach((el) => {
+      if (el.orderlist_id === product.orderlist_id) {
+        if (dataset.state === "+") {
+          el.quantity = parseInt(el.quantity) + 1;
+        } else if (dataset.state === "-") {
+          el.quantity = parseInt(el.quantity) - 1;
+        } else {
+          el.quantity = value;
+        }
+      }
+
+      myCartData.push(el);
+    });
+    this.setState({ myCartData });
+  };
```

Comment:
dataset과 value가 똑같아서 굳이 input에 data-state 속성을 안 써도 될 것 같네요!
```suggestion
   const { value } = e.target;
    const myCartData = [];
    this.state.myCartData.forEach(el => {
      if (el.orderList_id === product.orderList_id) {
        if (value === "+") {
          el.quantity = parseInt(el.quantity) + 1;
        } else if (value === "-") {
          el.quantity = parseInt(el.quantity) - 1;
        } else {
          el.quantity = value;
        }
      }

      myCartData.push(el);
    });
    this.setState({ myCartData });
```

Reply:
앗 너무 정신없이 하다보니 그렇내요!!!